gcexport.py

```diff
@@ -920,7 +992,7 @@ def main(argv):
                 details = None
                 tries = MAX_TRIES
                 while tries > 0:
-                    activity_details = http_req(URL_GC_ACTIVITY + str(actvty['activityId']))
+                    activity_details = http_req_as_string(URL_GC_ACTIVITY + str(actvty['activityId']))
```

Comment:
```suggestion
                    activity_details = http_req_as_string(f'{URL_GC_ACTIVITY}{actvty['activityId']}')
```

Reply:
As for Python2 my suggestion should be dropped and keep the original change from the PR